### PR TITLE
TDQ-15253 re-import strings from babili into studio: resolve some \n …

### PR DESCRIPTION
**Pull request by xqliu-talend**
…issues

### CODE REVIEW

File: i18n/plugins/org.talend.dataquality.edit.nl/plugin_hr.properties [hunk 1]
```diff
@@ -98,23 +98,11 @@ _UI_SAPFunctionUnit_tables_feature=Tablice
...
 _UI_SAPIDocUnit_xmlFile_feature=XML datoteka
 _UI_XmlFileConnection_XsdFilePath_feature=Path Xsd datoteke
-_UI_XmlFileConnection_schema_feature=
-
-	
-
-Collection
-
-Schema
+_UI_XmlFileConnection_schema_feature=Schema
```
Comment (scorreia) [inline i18n/plugins/org.talend.dataquality.edit.nl/plugin_hr.properties]: shouldn't it be Collection\nSchema?

File: i18n/plugins/org.talend.dataquality.edit.nl/plugin_hr.properties [hunk 2]
```diff
@@ -98,23 +98,11 @@ _UI_SAPFunctionUnit_tables_feature=Tablice
...
 _UI_XmlFileConnection_loop_feature=Petlja
 _UI_SchemaTarget_TagName_feature=Naziv Tag-a
-_UI_SchemaTarget_schema_feature=
-
-	
-
-Collection
-
-Schema
+_UI_SchemaTarget_schema_feature=Schema
```
Comment (scorreia) [inline i18n/plugins/org.talend.dataquality.edit.nl/plugin_hr.properties]: same as above

File: i18n/plugins/org.talend.dataquality.edit.nl/plugin_hr.properties [hunk 3]
```diff
@@ -138,13 +126,7 @@ _UI_Concept_LoopExpression_feature=Izraz Petlje
...
 _UI_Concept_group_feature=grupa
 _UI_Concept_loop_feature=Petlja
-_UI_ConceptTarget_schema_feature=
-
-	
-
-Collection
-
-Schema
+_UI_ConceptTarget_schema_feature=Schema
```
Comment (scorreia) [inline i18n/plugins/org.talend.dataquality.edit.nl/plugin_hr.properties]: again

File: i18n/plugins/org.talend.dataquality.edit.nl/plugin_hr.properties (hunk 1 above)
Comment (xqliu-talend) [reply i18n/plugins/org.talend.dataquality.edit.nl/plugin_hr.properties]: for other language, it just Schema, not Collection\nSchema, so for language "hr", it should be Schema also

**Comment by xqliu-talend** [comment]
let's focus on https://github.com/Talend/tdq-studio-se/pull/300

**Pull request closed without merging**